app/src/main/java/com/willowtree/vocable/presets/MySayingsEmptyFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.willowtree.vocable.presets
+
+import android.content.res.Configuration
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.view.isVisible
+import androidx.core.view.updateLayoutParams
+import com.willowtree.vocable.BaseFragment
+import com.willowtree.vocable.R
+import com.willowtree.vocable.databinding.MySayingsEmptyLayoutBinding
+import org.koin.android.ext.android.bind
+
+class MySayingsEmptyFragment : BaseFragment() {
+
+    private var binding: MySayingsEmptyLayoutBinding? = null
+
+    companion object {
+        private const val KEY_IS_SETTINGS = "KEY_IS_SETTINGS"
+
+        fun newInstance(isSettings: Boolean): MySayingsEmptyFragment {
+            return MySayingsEmptyFragment().apply {
+                arguments = Bundle().apply {
+                    putBoolean(KEY_IS_SETTINGS, isSettings)
+                }
+            }
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = MySayingsEmptyLayoutBinding.inflate(inflater, container, false)
+
+        arguments?.getBoolean(KEY_IS_SETTINGS)?.let { isSettings ->
+            when {
+                isSettings -> {
+                    binding?.star?.isVisible = true
+                    binding?.emptyMySayingsText?.setText(R.string.my_sayings_empty_settings)
+                }
+                !isSettings && resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE && !resources.getBoolean(R.bool.is_tablet) -> {
+                    val param = binding?.emptyMySayingsText?.layoutParams as ConstraintLayout.LayoutParams
+                    param.setMargins(0,0,0,0)
+                    binding?.emptyMySayingsText?.layoutParams = param
```

Comment:
Why this logic instead of just having a separate layout file?

Reply:
because if it is coming from settings it does need a margin. Is there a way for me to make two separate landscape layouts? that'd be dope :D

Reply:
Ok gotcha. This is cool then, not sure if there's an easy way to get two separate layouts